wallet/how-to/use-non-evm-networks/starknet/send-starknet-transactions.md

```diff
@@ -0,0 +1,66 @@
+---
+description: Send Starknet transactions in MetaMask.
+sidebar_position: 4
+---
+
+# Send Starknet transactions
+
+After the account is connected, you can send a transaction using the `starknet.invoke()` function:
```

Comment:
The `connect-to-starknet` section shows how to get access to the wallet object that we use here 

```suggestion
After the account is connected, you can send a transaction using the `wallet.account.execute` function:
```